tests/device_tests/test_reset_backup_bip39.py

```diff
@@ -0,0 +1,167 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2019 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+from unittest import mock
+
+import pytest
+from mnemonic import Mnemonic
+
+from trezorlib import device, messages
+from trezorlib.messages import BackupType, ButtonRequestType as B
+
+from ..common import click_through, generate_entropy, read_and_confirm_mnemonic
+
+EXTERNAL_ENTROPY = b"zlutoucky kun upel divoke ody" * 2
+OS_URANDOM = mock.Mock(return_value=EXTERNAL_ENTROPY)
+
+
+@pytest.mark.skip_t1
```

Comment:
Couldn't we run this on T1 as well?

Reply:
Here's a bit more complex. T1 and TT send different ButtonRequests during reset so we'd need separate input flow and expected responses for T1 and TT. I can have a look at it if it's something we want.

Reply:
I think let's move on for now.